sensors/inject/lidar.go

```diff
@@ -0,0 +1,40 @@
+// Package inject provides dependency injected structures for mocking interfaces.
```

Comment:
Refactoring: Split contents of `sensor_mock.go` into `inject/imu.go` and `inject/lidar.go` and changed the mock sensors into inject sensors (which they pretty much were already).